bluetooth/src/commonMain/kotlin/extensions/Accessors.kt

```diff
@@ -0,0 +1,42 @@
+package com.splendo.kaluga.bluetooth.extensions
+
+import com.splendo.kaluga.base.utils.bytesOf
+import com.splendo.kaluga.bluetooth.Characteristic
+import com.splendo.kaluga.bluetooth.UUIDException
+import com.splendo.kaluga.bluetooth.characteristics
+import com.splendo.kaluga.bluetooth.device.Device
+import com.splendo.kaluga.bluetooth.services
+import kotlinx.coroutines.flow.*
+
+/**
+ * Provides access to device data flow by service and characteristic string uuids.
+ * @param serviceUUID string service uuid representation
+ * @param characteristicUUID string characteristic uuid representation
+ * @throws UUIDException.InvalidFormat
+ */
+fun Flow<Device?>.dataFlow(serviceUUID: String, characteristicUUID: String) = characteristicsFlow(serviceUUID, characteristicUUID).flatMapConcat {
+    it.enableNotification()
```

Comment:
You have to mention this in comments. Also calling this data flow twice will enable notification twice?